recipes/onnx2torch/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "onnx2torch" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f668f6ecb2f34029d4c85a66be2f4a329a3d5d2ed8513395e508322d20dcb859
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - numpy >=1.16.4
+    - onnx >=1.9.0
+    - pip
+    - python >=3.6
+    - torchvision >=0.9.0
```

Unless the package imports itself at build time you don't need the run time dependencies on host.

```suggestion
    - pip
    - python >=3.6
```

---

:heavy_check_mark: